src/crafting.cpp

```diff
@@ -50,7 +49,7 @@ void remove_from_component_lookup(recipe* r);
 
 recipe::~recipe()
 {
-    remove_from_component_lookup(this);
+    recipe_dict.remove_recipe( this );
```

Comment:
Yes yes, I'm always on the fence on the naming of such functions. Thanks for the sanity check